service/init.go

```diff
@@ -23,7 +23,6 @@ import (
 	nrf_cache "github.com/omec-project/nrf/nrfcache"
 	"github.com/omec-project/openapi/models"
 	pfcpLogger "github.com/omec-project/pfcp/logger"
```

Comment:
Should not this line also be removed?
```suggestion
```

Reply:
Of course, with the appropriate changes inside the file, which uses this import.